start/account/AP7/ap7-access.md

```diff
@@ -0,0 +1,103 @@
+
+
+### Option 1: Login via Web Authentication
+
+Logging in via web authentication requires no preparatory steps beyond having access to an internet browser. 
+
+To authenticate using this approach: 
+
+1. Open a terminal and type `ssh username@ap7.ospool.osg-htc.org`, being sure to replace `username` with your `ap7.ospool.osg-htc.org` username. Upon hitting enter, the following text should appear with a unique, but similar, URL: 
+
+   ```
+   Authenticate at
+   -----------------
+   https://cilogon.org/device/?user_code=FF4-ZX6-9LK
+   -----------------
+   Type 'Enter' when you authenticate.
+   ```
+
+2. Copy the `https://` link, paste it into a web browser, and hit enter.  
+
+3. You will be redirected to a new page where you will be prompted to login using your institutional credentials. Once you have done so, a new page will appear with the following text: "You have successfully approved the user code. Please return to your device for further instructions."
+
+4. Return to your terminal, and type 'Enter' to complete the login process. 
+
+
+### Option 2: Login via SSH Key Pair Authentication
+
+It is also possible to authenticate using an SSH key pair, if you prefer. Logging in using SSH keys does not require access to an internet browser to login into the OSG Access Point, `ap7.ospool.osg-htc.org`. 
+
+The process below describes how to upload a public key to the registration website. It assumes that a private/public key pair has already been generated. If you need to generate a key pair, see the "Step 1: Generate SSH Keys" section of this [OSG guide](https://support.opensciencegrid.org/support/solutions/articles/12000027675-generate-ssh-keys-and-activate-your-osg-login). 
+
+1. Return to the [Registration Page](https://registry.cilogon.org/registry/co_petitions/start/coef:297) and login using your institutional credentials if prompted.
+
+1. Click your name at the top right. In the dropdown box, click "My Profile (OSG)" button.
+
+   {: .border .border-5}
+   <img src="ap7-images/ssh-homepage-dropdown.png" class= "img-fluid"/>
+
+1. On the right hand side of your profile, click "Authenticators" link.
+
+   {: .border .border-5 }
+   <img src="ap7-images/ssh-edit-profile.png" class= "img-fluid"/>
+
+1. On the authenticators page, click the "Manage" button.
+
+   {: .border .border-5 }
+   <img src="ap7-images/ssh-authenticator-select.png" class= "img-fluid"/>
+
+1. On the new SSH Keys page, click "Add SSH Key" and browse your computer to upload your public SSH key.
+   
+   {: .border .border-5 }
+   <img src="ap7-images/ssh-key-list.png" class= "img-fluid"/>
+
```

Comment:
I might add a note at the end, saying something like
> You can now log in to Access Point, `ap7.ospool.osg-htc.org` from the terminal, 
> using the same `ssh` command that was shown in the first part of the web authentication section. 
> When you log in, instead of being prompted with a web link, you should either authenticate 
> automatically or be asked for your ssh key passphrase to complete logging in.

Reply:
Good idea! 

Also, thank you for catching all of my terrible "login" grammer mistakes!